flexinput/src/main/res/values/styles.xml

```diff
@@ -119,6 +119,15 @@
         <item name="android:paddingBottom">@dimen/add_content_viewpager_bottom_margin</item>
         <item name="android:clipToPadding">false</item>
     </style>
+
+    <style name="FlexInput.AddContent.Base.Button" parent="FlexInput.AddContent.Base.FloatingActionButton">
```

Comment:
why is this called `Base.Button`? It should be more specific since it applies directly to a element. The pattern I'm trying to keep in this library is that `*.Base.*` is the default attributes and you just extend and add/override as needed.

Also since the button isn't a FAB, I don't think we should extend the `FloatingActionButton` style. Thus why I renamed the original and moved around some styles in the snippet I sent.